sotodlib/toast/instrument.py

```diff
@@ -471,6 +476,29 @@ def get_par_float(ddata, key, default):
             sample_rate=sample_rate,
         )
 
+    @classmethod
+    def _load_hdf5(
+        cls, handle, comm=None, detectors=None, file_det_sets=None, **kwargs
+    ):
+        """Load simulated focalplane from HDF5.
+
+        Although SOFocalplane inherits from toast.instrument.Focalplane, it
+        essentially is just a custom constructor and the actual content is
+        stored in the Focalplane base class.  When loading data we just
+        dispatch to the base class method.
+        """
+        return toast.instrument.Focalplane._load_hdf5(
```

Comment:
Obviously this will work fine but why not just call `SOFocalplane._load_hdf5()`?